<launch>
  <arg name="config_file" default="$(find-pkg-share autoware_pointcloud_merger)/config/pointcloud_merger.param.yaml" description="Path to the configuration YAML file"/>

  <group>
    <node pkg="autoware_pointcloud_merger" exec="autoware_pointcloud_merger_node" name="pointcloud_merger" output="screen">
      <param from="$(var config_file)" allow_substs="true"/>
    </node>
  </group>
</launch>
